### ISSUE TITLE
Value in Recessions and Recoveries | Research Affiliates

### ISSUE THREAD

**Issue by dawu76** [dawu76/investment-research-notes#9]
"In recoveries, low vol strategies slightly underperform, again, quite consistently in five out of six cases. Because the performance of these strategies deteriorates relatively less in a bear market than the performance of other strategies and because their performance lag in the recovery is relatively minimal, over the full cycle low vol strategies tend to outperform the market by a substantial margin (19.4%)."

The quality strategies show strong performance in recoveries, beating the market in all six out of six cases. The companies included in quality strategies tend to be more profitable and conservatively run. They tend to have deeper and broader moats that allow them to withstand a recessionary environment as well as to take advantage of opportunities that may present themselves during a recession.

During recoveries, small-cap stocks quite consistently outperform the large-cap market. The fears that keep investors from wanting to hold riskier small-cap stocks in recessions ultimately reward the investors who do own them as the market recovers (22.0%).

In summary, after examining six downturns and subsequent recoveries (which we should remember is a small sample!), we find that value, low volatility, quality, and small-cap strategies tend to do quite well in bear markets following the bursting of a bubble. If the bear market, however, is driven by a shock to fundamentals, only the low volatility strategy appears to offer significant performance protection. We conclude that recessionary periods can create opportunities for factor investors. In the recoveries that follow the recessions, value, quality, and small-cap strategies tend to perform quite well.
 https://www.researchaffiliates.com/publications/articles/808-value-in-recessions-and-recoveries.html#:~:text=In%20recoveries%2C%20low,substantial%20margin%20(19.4%25).
